Lib/concurrent/futures/process.py

```diff
@@ -505,9 +512,16 @@ def __init__(self, max_workers=None, mp_context=None,
 
         if max_workers is None:
             self._max_workers = os.cpu_count() or 1
+            if sys.platform == 'win32':
+                self._max_workers = min(_MAX_WINDOWS_WORKERS,
+                                        self._max_workers)
         else:
             if max_workers <= 0:
                 raise ValueError("max_workers must be greater than 0")
+            elif (sys.platform == 'win32' and
+                max_workers > _MAX_WINDOWS_WORKERS):
+                raise ValueError(
```

Comment:
Should we consider automatically changing max_workers to 61 instead? That would make for a nicer API for people using `max_workers=os.cpu_count()`, and it would make code like what was in Black before https://github.com/python/black/pull/838/files work unchanged.

On the other hand, it may be unexpected that we silently use fewer workers than the user asked for.

Reply:
I thought silently using fewer processes could be surprising. I also think that it's pretty unusual in the standard library to silently fix bad argument values.

Reply:
I like raising when too many workers are requested, and doing it quietly when it was "automatic".